xml/ay_bigfile.xml

```diff
@@ -16950,20 +16954,29 @@ echo -n 100
         </entry>
         <entry>
          <para>
-          No <literal>autoyast</literal> variable needed for
-          an automated upgrade, see <xref linkend="pro-upgrade-upgrade-mode"/>.
-         </para>
-        </entry>
-       </row>
-       <row>
-        <entry>
-         <para>
-          <literal>autoyast=default</literal>
+          Upgrade: No <literal>autoyast</literal> variable needed for
+          an automated offline upgrade, see <xref linkend="pro-upgrade-offline-upgrade-mode"/>.
          </para>
-        </entry>
-        <entry>
-         <para>
-          Default auto-installation option.
+	 <para>
+	   Installation: &ay; will be started if a file named <filename>autoinst.xml</filename> is in the following
+	   three places:
+           <orderedlist>
```

Comment:
Travis reports a build error because you need to close your para before starting the orderedlist. You can't have a list inside a paragraph. To see this error in your local build, run `daps  -d DC-SLES-autoyast validate`. If that does not report the error, you are probably validating with Docbook instead of Geekodoc. Let me know if you want some help with this.